generators/newapp/new.go

```diff
@@ -86,12 +68,25 @@ func (a Generator) Run(root string, data makr.Data) error {
 		return errors.WithStack(err)
 	}
 
-	if a.WithModules {
-		g.Add(makr.NewCommand(exec.Command(envy.Get("GO_BIN", "go"), "mod", "init", a.PackagePkg)))
+	if a.WithDep {
+		data["addPrune"] = true
+		g.Add(makr.NewFile("Gopkg.toml", GopkgTomlTmpl))
+		if _, err := exec.LookPath("dep"); err != nil {
+			// This step needs to be in a separate generator, because goGet() exec.Command
+			// checks if the executable exists (so before running the generator).
+			gg := makr.New()
+			gg.Add(makr.NewCommand(makr.GoGet("github.com/golang/dep/cmd/dep")))
+			if err := gg.Run(root, data); err != nil {
+				return errors.WithStack(err)
+			}
+		}
+	} else {
+		if _, err := exec.LookPath("goimports"); err != nil {
```

Comment:
If I use dep, will goimports still be installed?

Reply:
lol. fixed.